win/tty/termcap.c

```diff
@@ -1,4 +1,4 @@
-/* NetHack 3.6	termcap.c	$NHDT-Date: 1562056615 2019/07/02 08:36:55 $  $NHDT-Branch: NetHack-3.6 $:$NHDT-Revision: 1.31 $ */
+/* NetHack 3.7	termcap.c	$NHDT-Date: 1609454952 2020/12/31 22:49:12 $  $NHDT-Branch: NetHack-3.7 $:$NHDT-Revision: 1.40 $ */
```

Comment:
I'm assuming the fact this is NH 3.7 doesn't matter for our purposes.

Reply:
Yes this is a CVS/SVN string, see http://web.mit.edu/~xela/WWW/cvs-keywords.html